tests/e2e/specs/shopper/cart-checkout/checkout.test.js

```diff
@@ -44,16 +44,6 @@ describe( 'Shopper → Checkout', () => {
 		await shopper.block.emptyCart();
 	} );
 
-	it( 'User can view empty cart message', async () => {
```

Comment:
It seems that this test isn't necessary anymore (p1665157378161879-slack-C8X6Q7XQU)